lib/oembed/utils.py

```diff
@@ -7,12 +7,16 @@
 from htk.lib.oembed.constants import *
 from htk.utils.request import get_current_request
 
-def get_oembed_html(url):
+def get_oembed_html(url, autoplay=False):
     """Gets the oEmbed HTML for a URL, if it is an oEmbed type
     """
     oembed_type = get_oembed_type(url)
+
     if oembed_type:
-        html = get_oembed_html_for_service(url, oembed_type)
+        if oembed_type == "youtube":
```

Comment:
single quotes

Reply:
Is this the preferred style when doing strings?

Reply:
yup, always do single quotes unless some other reason, like needing to use an unescaped single quote.

but even in that situation, in Python I'd use triple quotes `"""`